integration_test.go

```diff
@@ -477,3 +648,44 @@ func getContractAddressesFromContractRegistry(ethHttpUrl string) ContractAddress
 		Erc20MockStrategy:      erc20MockStrategyAddr,
 	}
 }
+
+func NewEthHttpClient(rpcAddress string) (*ethclient.Client, error) {
+	client, err := ethclient.Dial(rpcAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	return client, nil
+}
```

Comment:
This is not very useful. The whole body can be replaced with `return ethclient.Dial(rpcAddress)`, which at this point what's the point of having a function like this? Since you'll have to check the errors outside anyways.

Reply:
https://github.com/Layr-Labs/avs-sync/pull/45/commits/96a4793d05e2ab635ee1eef0f3d6d73649a681a2